inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/thirdparty/sort/util/SinkInfoUtils.java

```diff
@@ -76,13 +80,53 @@ public static SinkInfo createSinkInfo(InlongStreamInfo inlongStreamInfo, SourceR
         String sinkType = sinkResponse.getSinkType();
         if (SinkType.forType(sinkType) == SinkType.HIVE) {
             return createHiveSinkInfo(sourceResponse, (HiveSinkResponse) sinkResponse);
-        } else if (SinkType.forType(sinkType) == SinkType.KAFKA) {
+        }
+        if (SinkType.forType(sinkType) == SinkType.KAFKA) {
```

Comment:
Please use `else if`, not all `if` statement.

Reply:
done